lib/checkPullRequestLabels.js

```diff
@@ -363,3 +365,41 @@ module.exports.checkForIssueLabel = async function (context) {
     await context.github.issues.removeLabel(removeLabelParams);
   }
 };
+
+/**
+ * This function checks and prevent the removal of stale build label
+ * by unauthorised users.
+ *
+ * @param {import('probot').Context} context
+ */
+module.exports.checkStaleBuildLabelRemoval = async function (context) {
+  /**
+   * @type {import('probot').Octokit.IssuesGetLabelResponse} label
+   */
+  const label = context.payload.label;
+  const pullRequest = context.payload.pull_request;
+  if (label.name === OLD_BUILD_LABEL) {
+    const user = context.payload.sender;
+    /*const {data: lastCommit} = await context.github.repos.getCommit(
+      context.repo({
+        ref: pullRequest.head.sha
+      })
+    );*/
+
+    // const lastCommitDate = new Date(lastCommit.commit.author.date);
+    // Commit is older than 2 days
```

Comment:
This comment can easily become stale, since the number of days depends on the `checkBuild` method.

Reply:
Done sir.